tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -120,7 +120,7 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
     p.Env['PROXY_CONFIG_HOSTDB_STORAGE_PATH'] = runtime_dir
     # p.Variables.RUNTIMEDIR=runtime_dir
     p.Setup.MakeDir(runtime_dir)
-    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True) 
+    p.Setup.Chown(runtime_dir, "nobody", "nobody",ignore=True)
```

Comment:
According to the config change, it should be under `runtime_dir/trafficserver`, as indicated by `storage_dir = os.path.join(runtime_dir, 'storage')` and line 140.